powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/SignatureSharedServiceBehavior.java

```diff
@@ -400,7 +400,8 @@ private void handleInvalidApplicationVersionImpl(ActivationRecordEntity activati
         }
 
         // Create the audit log record
-        auditingServiceBehavior.logSignatureAuditRecord(activation, signatureData, signatureType, false, activation.getVersion(),
+        final AuditingServiceBehavior.ActivationRecordDto activationDto = createActivationDtoFrom(activation);
```

Comment:
- [ ] I have to decide what to do with`setActivationStatus`. I prefer to audit the value before the change.

Reply:
Changed in a separate commit to better review.